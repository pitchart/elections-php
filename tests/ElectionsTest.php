<?php

namespace Tests\Elections\ElectionKata;

use PHPUnit\Framework\TestCase;
use function PHPUnit\Framework\assertTrue;

class ElectionsTest extends TestCase
{
    private $list = [
        "District 1" => ["Bob", "Anna", "Jess", "July"],
        "District 2" => ["Jerry", "Simon"],
        "District 3" => ["Johnny", "Matt", "Carole"],
    ];

    public function test_elections_without_district(): void
    {
        assertTrue(true);
    }

    public function test_elections_with_district(): void
    {
        assertTrue(true);
    }

}
