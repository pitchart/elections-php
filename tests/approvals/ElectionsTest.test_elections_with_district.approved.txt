[Michel] -> 0,00%
[Jerry] -> 66,67%
[Johnny] -> 33,33%
[Blank] -> 22,22%
[Null] -> 22,22%
[Abstention] -> 0,00%
